corehq/apps/cloudcare/templates/formplayer/debugger.html

```diff
@@ -89,18 +89,34 @@
     <!--ko if: xPathQuery.errorResult()-->
     <div data-bind="text: xPathQuery.errorResult(), css: 'text-danger'"></div>
     <!--/ko-->
-    <!--ko if: options.useCodeMirror && xPathQuery.successResult()-->
+    <!--ko if: xPathQuery.isSuccessTruncated() && options.showInFull-->
+    <div>
+        <i>Showing the first </i><i data-bind="text: xPathQuery.getMaxLines()"/><i> lines...</i>
+        <a class="clickable pull-right" data-clipboard-action="copy" data-bind="clipboardButton: xPathQuery.getFullSuccessResult()">
+            Copy Full Evaluation to Clipboard
```

Comment:
does the copied value contain valid XML? I thought that we removed whatever wrapper node was here. Probably fine, but might be weird if you paste it into a text editor and it gives you syntax errors.

Reply:
kind of, it's XML but not reliably a doc (Similar to what's there now).